### PR TITLE
Implement Pronunciation. Fix #27

### LINKED ISSUE #27: Pronunciation information
We are looking to add some pronunciation information to English WordNet and it would be good to add this as a schema extension. As I see it we would need to have the following information

* The actual form
* The notation scheme (e.g., IPA)
* The dialect, encoded with a ISO-3166 code
* Further notes. A free text for the describing the pronunciation in more detail

As such, I would suggest something like as follow:

```xml
<LexicalEntry id="ewn-transport-n">
  <Lemma writtenForm="transport" partOfSpeech="n">
    <Pronunciation notation="ipa" dialect="GB" notes="RP">/tɹænzˈpɔːt/</Pronunciation>
    <Pronunciation notation="ipa" dialect="GB" notes="RP">/tɹɑːnˈspɔːt/<Pronunciation>
    <Pronunciation notation="ipa" dialect="US" notes="GenAM">/tɹænzˈpɔɹt/</Pronunciation>
  </Lemma>
  <Sense>...</Sense>
</LexicalEntry>
```

### PR DESCRIPTION
**Pull request by jmccrae**

### CODE REVIEW

File: wn.ttl [hunk 1]
```diff
@@ -450,3 +450,29 @@
...
+  rdfs:domain :Pronunciation .
+
+:notation a owl:DatatypeProperty ;
+  rdfs:label "notation"@en ;
+  rdfs:comment "The notation scheme used for a pronunciation"@en ;
+  rdfs:domain :Pronunciation .
+
+:phonemic a owl:DatatypeProperty ;
+  rdfs:label "phonemic"@en ;
+  rdfs:comment "Indicates if the pronunciation is phonemic, in that represents every aspect of the pronunciation and does not omit information that could be assumed for this language"@en ;
```
Comment (goodmami) [inline wn.ttl:472]: I think this description is for "phonetic" (see [here](http://australianlinguistics.com/speech-sounds/phonemic-vs-phonetic/) for some explanation). In short, phonetic transcriptions may describe a particular utterance or a highly-specific transcription whereas phonemic transcriptions rely somewhat on the phonology of the language to reduce the level of detail. From the linked article, a phonemic transcription of "clean" would be /klin/ while a phonetic one might be [kl̥i:n]. For wordnets a phonemic transcription is probably more useful so the default value of "true" for the "phonemic" property (in the DTD) seems appropriate.

Comment (jmccrae) [reply wn.ttl]: Quite, right on the definition. I think the default value for phonemic is `true` in the DTD, right?

Comment (goodmami) [reply wn.ttl]: > I think the default value for phonemic is true in the DTD, right?

That's what I see in this PR, and I agree that it's a good default. But really (with apologies for the flip-flopping) I'm reconsidering having the delimiters `/.../` and `[...]` in the actual transcription. Originally I suggested not having the `/` slashes when I assumed we only cared about phonemic transcriptions, but if we want to allow phonetic ones (and possibly non-IPA transcriptions for which the phonetic/phonemic difference is unstated), why not just drop the attribute and allow such delimiters? It's shorter and probably clearer to someone who knows IPA.

(should I bring this discussion back to #27?)

Comment (jmccrae) [reply wn.ttl]: Yes, possibly best to discuss this on the issue

**Pull request merged**